roles/ciao-image/templates/ciao-image.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=CIAO Image Service
+After=network.target
+
+[Service]
+Type=simple
+ExecStart={{ bindir }}/ciao-image -identity=https://{{ keystone_fqdn }}:35357 \
```

Comment:
identity_service is configured in ciao configuration
https://github.com/01org/ciao/tree/master/configuration

Why should we pass it as an argument in the cmdline?
and where are the binary reading the certificates from ?